gossip/agent.go

```diff
@@ -106,6 +106,17 @@ func NewAgent(conf *Config, p []Processor) (agent *Agent, err error) {
 
 	return agent, nil
 }
+func chTimedSend(batch *protocol.BatchSnapshots, ch chan *protocol.BatchSnapshots) {
+	for {
+		select {
+		case <-time.After(200 * time.Millisecond):
+			log.Infof("Timed out sending out batch ")
```

Comment:
"Timed out enqueueing batch in out channel"

Reply:
Fixed